tda/mobile_native/dotnet-maui-android/test_checkout_dotnet_maui_android.py

```diff
@@ -0,0 +1,28 @@
+import time
+import sentry_sdk
+from appium.webdriver.common.appiumby import AppiumBy
+
+def test_checkout_dotnet_maui_android(android_dotnet_maui_driver):
+    driver = android_dotnet_maui_driver
+
+    try:
+        driver.find_element(AppiumBy.ID, "io.sentry.dotnet.maui.empowerplant:id/Real_World_Sample").click()
```

Comment:
Renamed:
https://github.com/sentry-demos/dotnet-maui/blob/16e03f7016b8d8355a01c56f8b14acc6c639118f/MainPage.xaml#L29

```suggestion
        driver.find_element(AppiumBy.ID, "io.sentry.dotnet.maui.empowerplant:id/Shop_For_Plants").click()
```